api/common_args_responses.go

```diff
@@ -58,6 +58,12 @@ type GetHeightResponse struct {
 	Height avajson.Uint64 `json:"height"`
 }
 
+type GetEpochResponse struct {
+	Number       avajson.Uint64 `json:"number"`
+	StartTime    avajson.Uint64 `json:"startTime"`
+	PChainHeight avajson.Uint64 `json:"pChainHeight"`
+}
```

Comment:
Should we include the epoch duration here as well, so that callers can determine how much time is left in the given epoch?